qp_shogun/sortmerna/__init__.py

```diff
@@ -0,0 +1,18 @@
+from qiita_client import QiitaCommand
+
+from .sortmerna import sortmerna
+
+__all__ = ['sortmerna']
+
+# Defining the Sortmerna command
+req_params = {'input': ('artifact', ['per_sample_FASTQ'])}
+opt_params = {
+    # Threads used
+    'Number of threads': ['integer', '5']
```

Comment:
Could you add the other SMR parameters, even if using defaults?

Reply:
Added.